Backpack/Theme/Classes/BPKDohaTheme.m

```diff
@@ -98,6 +98,18 @@ - (UIColor *)buttonSecondaryBorderColor {
     return BPKColor.gray100;
 }
 
+- (UIColor *)buttonDestructiveContentColor {
+    return [UIColor colorWithRed:0.368627451 green:0.02745098 blue:0.17254902 alpha:1];
```

Comment:
out of interest where did you get these colours? Just because they're so specific

Reply:
I colour picked them from the docs 😆